mu-plugins/global-fonts/style.css

```diff
@@ -1233,3 +1233,16 @@
 	src: url(./IBMPlexSans/IBMPlexSans-SemiBold-arrows.woff2) format("woff2");
 	unicode-range: U+2190-2199;
 }
+
+/*------------------------------*
+ *  Noto Serif JP               *
+ *------------------------------*/
+
+/* cjk */
+@font-face {
+	font-family: "Noto Serif JP";
+	font-weight: 200 900;
+	font-style: normal;
+	font-display: swap;
+	src: url(./Noto Serif/NotoSerifJP-cjk.woff2) format("woff2");
```

Comment:
I think it's the same as @t-hamano's comment. I have to set this to 
```
./Noto\ Serif/NotoSerifJP-cjk.woff2
``` 
to make it work. Otherwise, the font isn't applied.